### PR TITLE
Abdul/rfid for io t

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for RFIDForIoT app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : Mfrc522.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : Buzzer.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

3) file modified : RFIDForIoT.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

4) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

**Review by rsameser — APPROVED** [review]

File: RFIDForIoT/CS/Mfrc522Lib/Mfrc522.cs [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+namespace Mfrc522Lib
+{
+    public class Mfrc522
+    {
+        public SpiDevice _spi { get; private set; }
+        public GpioController IoController { get; private set; }
+        public GpioPin _resetPowerDown { get; private set; }
+        public int pincount;
+
+        private const string SPI_CONTROLLER_NAME = "SPI2";
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/Mfrc522Lib/Mfrc522.cs]: SPI_CONTROLLER_NAME isn't used anywhere.  Remove it?

File: RFIDForIoT/CS/PiezoBuzzerLib/Buzzer.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Devices.Gpio;
+
+namespace PiezoBuzzerLib
+{
+    public class Buzzer
+    {
+        public const int PIEZO_PIN = 24; // CONNECT TO 32th PIN OF upboard
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/PiezoBuzzerLib/Buzzer.cs]: "32nd"  not  "32th"
"UP Board" not "upboard"

File: RFIDForIoT/CS/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+page_type: sample
+urlFragment: RFIDForIoT
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise 
+description: Sample to show how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/README.md]: Beep, Buzzer and Scans should not be capitalized.

File: RFIDForIoT/CS/README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise 
+description: Sample to show how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans
+---
+
+# RFID scanner with Windows 10 IoT Enterprise
+
+In this sample, we will demonstrate how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans.
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/README.md]: Beep, Buzzer and Scans should not be capitalized.

File: RFIDForIoT/CS/README.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+## Step 1: Load the project in Visual Studio
+
+## Step 2: Connect the MFRC522 to your Windows 10 IoT Enterprise device
+
+You'll need a few components:
+
+* MFRC522 scanner
+* a Piezo Buzzer (If you want a beep when a card scans)
+* a breadboard and a couple of connetor wires
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/README.md]: connector spelled incorrectly.

File: RFIDForIoT/CS/README.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+1. Connect RFID SDA to Pin 24
+2. Connect RFID SCK to Pin 23
+3. Connect RFID MOSI to Pin 19
+4. Connect RFID MISO to Pin 21
+5. Connect RFID GND to Pin 6
+6. Connect RFID RST to Pin 38
+7. Connect RFID 3.3V to Pin 1 (For some higher frequency cards this might need 5V)
+8. Connect Piezo Buzzer '+' to Pin 32
+9. Connect Piezzo Buzzer '-' to Pin 6
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/README.md:26]: Piezzo is spelled incorrectly, should be Piezo.

File: RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="d6bbe866-57dc-4016-ba7a-cdbdaed04cd6"
+    Publisher="CN=RFIDForIoT"
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest]: Publisher should be "CN=MSFT"

File: RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest [hunk 8]
```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="d6bbe866-57dc-4016-ba7a-cdbdaed04cd6"
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest]: Name="RFIDForIoT"

File: RFIDForIoT/CS/RFIDForIoT/RFIDForIoT.csproj [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+    <AssemblyName>RFIDForIoT</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.19041.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.15063.0</TargetPlatformMinVersion>
+    <MinimumVisualStudioVersion>14</MinimumVisualStudioVersion>
+    <FileAlignment>512</FileAlignment>
+    <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
+    <PackageCertificateKeyFile>RFIDForIoT_TemporaryKey.pfx</PackageCertificateKeyFile>
```
Comment (neilsh-msft) [inline RFIDForIoT/CS/RFIDForIoT/RFIDForIoT.csproj:20]: .pfx file is missing from pull request?

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: RFIDForIoT/CS/PiezoBuzzerLib/Buzzer.cs (hunk 2 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/PiezoBuzzerLib/Buzzer.cs]: Updated the changes

File: RFIDForIoT/CS/README.md (hunk 3 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/README.md]: Updated the Changes

File: RFIDForIoT/CS/README.md (hunk 4 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/README.md]: Updated the Changes

File: RFIDForIoT/CS/README.md (hunk 5 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/README.md]: Updated the Changes

File: RFIDForIoT/CS/README.md (hunk 6 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/README.md:26]: Updated the Changes

File: RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest (hunk 8 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest]: Changed the Name.

File: RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest (hunk 7 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/RFIDForIoT/Package.appxmanifest]: Changed the name

File: RFIDForIoT/CS/RFIDForIoT/RFIDForIoT.csproj (hunk 9 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/RFIDForIoT/RFIDForIoT.csproj:20]: image

.pfx filw is present in the pull request

File: RFIDForIoT/CS/Mfrc522Lib/Mfrc522.cs (hunk 1 above)
Comment (v-abdrahman) [reply RFIDForIoT/CS/Mfrc522Lib/Mfrc522.cs]: Available SPI devices gets recognize automatically in UPBOARD
SpiDevice.GetDeviceSelector();

Will remove this line

**Pull request merged**